cpp/src/io/parquet/writer_impl.cu

```diff
@@ -884,6 +879,9 @@ void writer::impl::write_chunk(table_view const &table, pq_chunked_state &state)
     // GPU column description
     auto *desc             = &col_desc[i];
     *desc                  = gpu::EncColumnDesc{};  // Zero out all fields
+    desc->column_data_base = col.data();
```

Comment:
Why did these come back?

Reply:
These members belong to stats_column_desc and are used for stats calculation in statistics/column_stats.cu::GatherColumnStatistics(). That function is called by ORC as well. If I remove those members/set them to nullptr then calls to GatherColumnStatistics will throw a cudaErrorIllegalAddress exception.